modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties

```diff
@@ -296,11 +296,13 @@ category-display-page-help=The display page is used to display the category.
 category-display-pages=Category Display Pages
 category-selected=Category Selected
 category.catalog=Catalog
+category.data-integration=Data Integration
 category.google-merchant=Google Merchant
 category.inventory=Inventory
 category.orders=Orders
 category.payment=Payment
 category.pricing=Pricing
+category.salesforce-talend-job-deployer=Salesforce Connector
```

Comment:
@dsitu Is correct to have label diffeent than the text?

Reply:
@marco-leo andrea-ale-sbarra hmm, I think we should try to keep them the same if we can.
So `category.salesforce-talend-job-deployer=Salesforce Talend Job Deployer` or `category.salesforce-connector=Salesforce Connector`

Reply:
let's go with category.salesforce-connector=Salesforce Connector